axelarnetwork/axelar-core
# feat(evm): add query for erc20 deposit status

## Description
Currently it is hard for users to determine if their erc20 deposit was actually confirmed, as a typo would result in their poll being discarded, and then they are unknowingly waiting for a 24 hour consolidation period to pass. This PR adds a query to check if the confirm erc20 deposit was successful

## Todos


## Steps to Test
First make a deposit and create a confirmation poll
axelard tx evm confirm-erc20-deposit ethereum {txID} {amount} {deposit addr} --from validator -y -b block

then check if the confirmation was successful
axelard q evm deposit-state ethereum [txID] [deposit address]

## Expected Behaviour
Returns the status of an erc20 deposit transaction confirmation


## Other Notes
